projects/rocprofiler-sdk/source/lib/python/rocpd/summary.py

```diff
@@ -363,12 +363,28 @@ def create_summary_region_queries(
     query = "SELECT DISTINCT(category) FROM regions_and_samples"
     categories = execute_statement(connection, query).fetchall()
 
+    category_prefixes = ["rocm_"]
+
     if region_categories is None:
         # Automatically retrieve region categories from the database
-        region_categories = set([cat[0].split("_")[0] for cat in categories])
+        region_categories = set()
+        for cat in categories:
+            category_name = cat[0]
+            matching_prefix = next(
+                (
+                    prefix
+                    for prefix in category_prefixes
+                    if category_name.lower().startswith(prefix.lower())
+                ),
+                "",
+            )
+            first_part = category_name[len(matching_prefix) :].split("_")[0]
+            region_categories.add(f"{matching_prefix}{first_part}")
 
     category_map = {
-        cat.lower(): [c[0] for c in categories if c[0].startswith(cat + "_")]
+        cat.lower(): [
+            c[0] for c in categories if c[0] == cat or c[0].lower().startswith(cat.lower() + "_")
```

Comment:
```suggestion
            c[0] for c in categories if c[0].lower() == cat.lower() or c[0].lower().startswith(cat.lower() + "_")
```
I think we should also compare lower case when they are single-word comparisons too.

Reply:
Done.